common/ConfigUtil.cpp

```diff
@@ -39,18 +39,65 @@ RuntimeConstant<bool> SslTermination;
 // 2) in the 'default' attribute in coolwsd.xml, which is for documentation
 // 3) the default parameter of getConfigValue() call. That is used when the
 //    setting is present in coolwsd.xml, but empty (i.e. use the default).
+// NOTE: This is sorted, please keep it sorted as it's friendlier to readers.
 static const std::map<std::string, std::string> DefAppConfig = {
     { "accessibility.enable", "false" },
-    { "allowed_languages", "de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru" },
     { "admin_console.enable_pam", "false" },
+    { "admin_console.logging.admin_action", "true" },
+    { "admin_console.logging.admin_login", "true" },
+    { "admin_console.logging.metrics_fetch", "true" },
+    { "admin_console.logging.monitor_connect", "true" },
+    { "allowed_languages", "de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru" },
+    { "browser_logging", "false" },
+    { "certificates.database_path", "" },
     { "child_root_path", "jails" },
-    { "file_server_root_path", "browser/.." },
+    { "deepl.api_url", "" },
+    { "deepl.auth_key", "" },
+    { "deepl.enabled", "false" },
+    { "document_signing.enable", "true" },
     { "enable_websocket_urp", "false" },
-    { "hexify_embedded_urls", "false" },
     { "experimental_features", "false" },
-    { "logging.protocol", "false" },
+    { "extra_export_formats.impress_bmp", "false" },
+    { "extra_export_formats.impress_gif", "false" },
+    { "extra_export_formats.impress_png", "false" },
+    { "extra_export_formats.impress_svg", "false" },
+    { "extra_export_formats.impress_swf", "false" },
+    { "extra_export_formats.impress_tiff", "false" },
+#if ENABLE_FEATURE_LOCK
+    { "feature_lock.calc_unlock_highlights", CALC_UNLOCK_HIGHLIGHTS },
+    { "feature_lock.draw_unlock_highlights", DRAW_UNLOCK_HIGHLIGHTS },
+    { "feature_lock.impress_unlock_highlights", IMPRESS_UNLOCK_HIGHLIGHTS },
+    { "feature_lock.is_lock_readonly", "false" },
+    { "feature_lock.locked_commands", LOCKED_COMMANDS },
+    { "feature_lock.locked_hosts.fallback[@disabled_commands]", "false" },
+    { "feature_lock.locked_hosts.fallback[@read_only]", "false" },
+    { "feature_lock.locked_hosts.host[0]", "localhost" },
+    { "feature_lock.locked_hosts.host[0][@disabled_commands]", "false" },
+    { "feature_lock.locked_hosts.host[0][@read_only]", "false" },
+    { "feature_lock.locked_hosts[@allow]", "false" },
+    { "feature_lock.unlock_description", UNLOCK_DESCRIPTION },
+    { "feature_lock.unlock_link", UNLOCK_LINK },
+    { "feature_lock.unlock_title", UNLOCK_TITLE },
+    { "feature_lock.writer_unlock_highlights", WRITER_UNLOCK_HIGHLIGHTS },
+#endif
+    { "file_server_root_path", "browser/.." },
+    { "help_url", HELP_URL }, // !MOBILEAPP
```

Comment:
iOS CI failed on this, missing #if !MOBILEAPP / #endif

Reply:
Yes, fixed now. Thanks.